Dockerfile.aarch64

```diff
@@ -23,7 +23,7 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM aarch64/ubuntu:trusty
+FROM aarch64/ubuntu:vivid
```

Comment:
The plan is switch to debian:jessie, see https://github.com/docker/docker/pull/20288